### PR TITLE
Center list of experiments regardless of screen size (closes #1131).

### LINKED ISSUE #1113: Need a way to get good content into staging & dev servers
Carrying over from PR #1111: For QA and general use of the staging & dev servers, we need a way to get good-enough content into them. Some options:
- a periodic content dump from prod to stage to dev? 
- apply `fixtures/demo_data.json`?

In either case, there's a big gotcha: Local development uses the local filesystem to host media like images - so we can do `cp -r fixtures/media/demo media`. But, on staging & dev, all media resources are hosted in Amazon S3 - so we have to upload all the related files to the right paths in the appropriate buckets.

This will need some ops assistance to a) import content into the database and b) get media resources into the S3 bucket. We don't have access to do either of those things on the dev team.

A prod content dump can also be tricky if it includes PII, but we can filter that out and/or be safer once we resolve #1034 and delete the bulk of user data (except our own as admins & contributors)

Ideally, we could script this to happen periodically with deployments.

### LINKED ISSUE #1131: Weird layout gap with even number of experiments in a row
Found in http://testpilot.dev.mozaws.net/__version__ (e9e40d0052b61faa8017abf8f4716dd947d24039):

``` json
{ "commit": "e9e40d0052b61faa8017abf8f4716dd947d24039",
  "source": "https://github.com/mozilla/testpilot",
  "version": "",
  "build": "https://circleci.com/gh/mozilla/testpilot/647" }
```

---
### Actual results:

firefox_test_pilot

### PR DESCRIPTION
**Pull request by chuckharmston**
r? @johngruen

### CODE REVIEW

**Comment by chuckharmston** [comment]
Tiny update to restore the original margin, as shown here:

screen shot 2016-08-10 at 12 20 32 pm

File: testpilot/frontend/static-src/styles/modules/_experiment-list.scss [hunk 1]
```diff
@@ -22,7 +26,7 @@
   border-radius: $small-border-radius + 1;
   box-shadow: 0 0 0 1px $transparent-black-1, 0 1px 0 1px $transparent-black-1;
   color: $black;
-  margin-bottom: $grid-unit * 1.75;
+  margin: 0 0 ($grid-unit * 1.75) $grid-unit;
```
Comment (johngruen) [inline testpilot/frontend/static-src/styles/modules/_experiment-list.scss]: @chuckharmston make the left margin here $grid-unit*2, then merge away!

**Comment by lmorchard** [comment]
Tests are failing for reasons unrelated to this PR. I'm working on a fix for the tests, but feel free to merge in the meantime

**Comment by chuckharmston** [comment]
Thanks @lmorchard, was just about to dig into the failures.

**Comment by ckprice** [comment]
I think this was meant to close #1131.

**Comment by chuckharmston** [comment]
Yup, thanks @ckprice!

**Comment by SoftVision-PaulOiegas** [comment]
Experiment tiles are correctly centered now even if the page is zoomed in, out or window is resized. Funny how the tiles resize. :smile:

**Pull request merged**